.github/workflows/golangci-lint.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v3
         with:
-          go-version: 1.17
+          go-version: 1.21
```

Comment:
golangci-lint is failing, I guess is related to this https://github.com/golangci/golangci-lint/issues/3933#issuecomment-1674808789

Reply:
Updated golangci-lint to 1.54.2. This seems to work as per https://github.com/golangci/golangci-lint/issues/3107#issuecomment-1744825702